### PR TITLE
Feat: Partners Section

### PR DESCRIPTION
**Pull request by mhenriette**
## Describe your changes
added partners section and made it responsive to all screens 
## Issue ticket number and link
- ID: 014
- Link: [Notion](https://www.notion.so/apeunit/Partners-Section-ecc390f75f76414ca7400157b03a179c)

## Tasks completed
- [x] add Partners Section
 
## Tasks not completed
- None

### CODE REVIEW

**Review by Ntwali-Josue — APPROVED** [review]
Well Done @mhenriette 🥇

**Review by Delyc — APPROVED** [review]
Looks good!

File: components/sections/Partners.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import Title from '../ui/Title'
+import Image from 'next/image';
+import logo from '../../data/logos'
+const Partners = () => {
+    return (
+        < div className='flex flex-col items-center  justify-center '>
```
Comment (alain-kubwayo) [inline components/sections/Partners.jsx:6]: You should remove empty spaces in the classes

File: components/sections/Partners.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            <Title content='Partners'/>
+           <p className='text-secondary-200 -mt-12  text-center'>We partner with the  leading brands</p> 
+            <div className='md:flex flex-wrap justify-center items-center hidden lg:mx-60'>
+            {
+                logo.map((el, index)=>{return <Image priority width={el.Width} height={el.Height} src={el.source} alt='logo' key={index} className='mx-3 my-2'/>
+                })
+            }
+            </div>
+            <div className='flex flex-wrap md:hidden justify-center w-full '>
```
Comment (alain-kubwayo) [inline components/sections/Partners.jsx:15]: The last space should be removed to stay consistent

**Review by alain-kubwayo — CHANGES_REQUESTED** [review]
Good job, I left some comments.

File: components/sections/Partners.jsx (hunk 1 above)
Comment (mhenriette) [reply components/sections/Partners.jsx]: done

File: components/sections/Partners.jsx (hunk 2 above)
Comment (mhenriette) [reply components/sections/Partners.jsx]: done

**Review by alain-kubwayo — APPROVED** [review]
Looks good!

**Pull request merged**